samples/UniversalApplication/ViewModels/PlayerViewModel.cs

```diff
@@ -61,7 +66,13 @@ public PlayerViewModel(IGrooveClient grooveClient, GrooveApiErrorViewModel error
 
             if (!string.IsNullOrEmpty(streamResponse.Url))
             {
-                StreamUrl = streamResponse.Url;
+                Uri streamResponseUri = new Uri(streamResponse.Url);
+
+                MediaSource =
+                    streamResponse.ContentType == "application/vnd.apple.mpegurl"
+                        ? MediaSource.CreateFromAdaptiveMediaSource(
+                            (await AdaptiveMediaSource.CreateFromUriAsync(streamResponseUri)).MediaSource)
```

Comment:
>(await AdaptiveMediaSource.CreateFromUriAsync(streamResponseUri)).MediaSource) [](start = 28, length = 78)

in the sample codes we saw this morning on the interwebs everyone seemed to check if the AdaptiveMediaSource had a success status before passing it to the MediaSource, do you know if that's important?